src/sas/qtgui/Perspectives/Inversion/InversionPerspective.py

```diff
@@ -494,6 +501,41 @@ def updateDataList(self, dataRef):
         if self.batchResultsWindow is not None:
             self.showBatchOutput()
 
+    def getState(self):
+        """
+        Collects all active params into a dictionary of {name: value}
+        :return: {name: value}
+        """
+        params = {
+            'alpha': self._calculator.alpha,
+            'background': self._calculator.background,
+            'chi2': self._calculator.chi2,
+            'cov': self._calculator.cov,
+            'd_max': self._calculator.d_max,
+            'elapsed': self._calculator.elapsed,
+            'err': self._calculator.err,
+            'est_bck': self._calculator.est_bck,
+            'iq0': self._calculator.iq0(self._calculator.out),
+            'nerr': self._calculator.nerr,
+            'nfunc': self.getNFunc(),
+            'npoints': self._calculator.npoints,
+            'ny': self._calculator.ny,
+            'out': self._calculator.out,
+            'oscillations': self._calculator.oscillations(self._calculator.out),
+            'pos_frac': self._calculator.get_positive(self._calculator.out),
+            'pos_err': self._calculator.get_pos_err(self._calculator.out,
+                                                    self._calculator.cov),
+            'q_max': self._calculator.q_max,
+            'q_min': self._calculator.q_min,
+            'rg': self._calculator.rg,
+            'slit_height': self._calculator.slit_height,
+            'slit_width': self._calculator.slit_width,
+            'suggested_alpha': self._calculator.suggested_alpha,
+            'x': self._calculator.x,
+            'y': self._calculator.y,
+        }
+        return params
```

Comment:
apart of ease of debugging, there's no reason to introduce a separate variable `params`. You could just create the dictionary as the argument of `return`.